src/hooks/useLauncherClient.ts

```diff
@@ -9,17 +9,17 @@ interface useLauncherClientReturn {
   launcherClient?: CozyClient
 }
 export const useLauncherClient = (
-  launcherContext?: LauncherContext
+  launcherContextValue?: LauncherContext['value']
```

Comment:
Why not exporting `LauncherContextValue` from `models`?

Reply:
indeed, will update

Reply:
✔️